lib/eventboss/error_handlers/sentry.rb

```diff
@@ -0,0 +1,19 @@
+module Eventboss
+  module ErrorHandlers
+    class Sentry
+      def call(exception, context = {})
+        eventboss_context = { component: 'eventboss' }
+        eventboss_context[:action] = context[:processor].class.to_s if context[:processor]
+
+        default_options = { }
+
+        ::Sentry.with_scope do |scope|
+          scope.set_tags(
+            context.merge(eventboss_context, default_options)
```

Comment:
I would remove `default_options` as it adds no value to merge an empty Hash :)

Reply:
True, I would rather do this:

```suggestion
            eventboss_context.merge(context)
```
But let's do like this to have the same way as it was in rollbar:

```suggestion
            context.merge(eventboss_context)
```